how-to/offline_cpu.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Description: Offline SOS CPUs except BSP before launch UOS
+
+for i in `ls -d /sys/devices/system/cpu/cpu[1-9]*`; do
```

Comment:
This needs to be the following as you are ignoring the first CPU:

```sh
for i in `ls -d /sys/devices/system/cpu/cpu[0-9]*`; do
```

Reply:
We don't want to check cpu0. So [1-9]*. 

With [0-9]* get the following error
`cat: /sys/devices/system/cpu/cpu0/online: No such file or directory`